lambda/exportPass/index.js

```diff
@@ -0,0 +1,138 @@
+const AWS = require('aws-sdk');
+const s3 = new AWS.S3();
+const csvjson = require('csvjson');
+const { runQuery } = require('../dynamoUtil');
+const { sendResponse } = require('../responseUtil');
+const { checkPermissions } = require('../permissionUtil');
+
+exports.handler = async (event, context) => {
+  console.log('Export Pass', event);
+  console.log('event.queryStringParameters', event.queryStringParameters);
+
+  let queryObj = {
+    TableName: process.env.TABLE_NAME
+  };
+
+  try {
+    if (!event.queryStringParameters) {
+      return sendResponse(400, { msg: 'Invalid Request' }, context);
+    }
+    if (event.queryStringParameters.facilityName && event.queryStringParameters.park) {
+
+      const tokenObj = await checkPermissions(event);
+      if (tokenObj.decoded === false) {
+        return sendResponse(403, { msg: 'Unauthorized' });
+      }
+      // Get all the passes for a specific facility
+      queryObj.ExpressionAttributeValues = {};
+      queryObj.ExpressionAttributeValues[':pk'] = { S: 'pass::' + event.queryStringParameters.park };
+      queryObj.ExpressionAttributeValues[':facilityName'] = { S: event.queryStringParameters.facilityName };
+      queryObj.KeyConditionExpression = 'pk =:pk';
+      queryObj.FilterExpression = 'facilityName =:facilityName';
+
+      // Filter Date
+      if (event.queryStringParameters.date) {
+        const theDate = new Date(event.queryStringParameters.date);
+        const dateselector = theDate.toISOString().split('T')[0];
```

Comment:
Not much different here but a few other endpoints are using date-fns:

```
const { formatISO } = require('date-fns');
...
const dateselector = formatISO(theDate, { representation: 'date' });
```

Reply:
Fixed in b07f491